glue/src/main/scala/com/gu/tableversions/glue/GlueMetastore.scala

```diff
@@ -0,0 +1,184 @@
+package com.gu.tableversions.glue
+
+import java.net.URI
+
+import cats.effect.Sync
+import cats.implicits._
+import com.amazonaws.services.glue.AWSGlue
+import com.amazonaws.services.glue.model.{
+  CreatePartitionRequest,
+  DeletePartitionRequest,
+  GetPartitionsRequest,
+  GetPartitionsResult,
+  GetTableRequest,
+  PartitionInput,
+  StorageDescriptor,
+  TableInput,
+  UpdatePartitionRequest,
+  UpdateTableRequest,
+  Partition => GluePartition,
+  Table => GlueTable
+}
+import com.gu.tableversions.core.Partition.{ColumnValue, PartitionColumn}
+import com.gu.tableversions.core.{PartitionedTableVersion, SnapshotTableVersion, TableName, TableVersion, _}
+import com.gu.tableversions.metastore.Metastore.TableOperation
+import com.gu.tableversions.metastore.Metastore.TableOperation.{
+  AddPartition,
+  RemovePartition,
+  UpdatePartitionVersion,
+  UpdateTableVersion
+}
```

Comment:
`import ... ._` ... (sorry!)

Reply:
Yeah I'll fix all of this, I just expected intellij "optimise imports" to do this on its own but I guess it doesn't